cmd/pitr/collector/collector.go

```diff
@@ -10,6 +10,7 @@ import (
 	"os/exec"
 	"strings"
 	"syscall"
+	"time"
```

Comment:
Line 29
```
bufferSize     int64  // size of uploading buffer
```
is not used

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/723/commits/7476cb110a90a023d52b522c6ab1014e94ad895d